src/main/java/io/milvus/v2/client/MilvusClientV2.java

```diff
@@ -0,0 +1,485 @@
+package io.milvus.v2.client;
+
+import io.milvus.grpc.MilvusServiceGrpc;
+import io.milvus.grpc.MutationResult;
+import io.milvus.param.LogLevel;
+import io.milvus.param.R;
+import io.milvus.param.RpcStatus;
+import io.milvus.v2.service.collection.request.*;
+import io.milvus.v2.service.collection.response.DescribeCollectionResp;
+import io.milvus.v2.service.collection.response.ListCollectionsResp;
+import io.milvus.v2.service.index.request.CreateIndexReq;
+import io.milvus.v2.service.index.request.DescribeIndexReq;
+import io.milvus.v2.service.index.request.DropIndexReq;
+import io.milvus.v2.service.index.response.DescribeIndexResp;
+import io.milvus.v2.service.partition.request.*;
+import io.milvus.v2.service.rbac.request.*;
+import io.milvus.v2.service.rbac.response.DescribeRoleResp;
+import io.milvus.v2.service.rbac.response.DescribeUserResp;
+import io.milvus.v2.service.utility.request.AlterAliasReq;
+import io.milvus.v2.service.utility.request.CreateAliasReq;
+import io.milvus.v2.service.utility.request.DropAliasReq;
+import io.milvus.v2.service.utility.request.FlushReq;
+import io.milvus.v2.service.vector.request.*;
+import io.milvus.v2.service.vector.response.GetResp;
+import io.milvus.v2.service.vector.response.QueryResp;
+import io.milvus.v2.service.vector.response.SearchResp;
+import lombok.NonNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class MilvusClientV2 extends BaseClient {
+
+    protected static final Logger logger = LoggerFactory.getLogger(MilvusClientV2.class);
+    protected LogLevel logLevel = LogLevel.Info;
+
+    public MilvusClientV2() {
+
+    }
+    /**
+     * Creates a Milvus client instance.
+     *
+     * @param connectConfig Milvus server connection configuration
+     */
+    public void connect(ConnectConfig connectConfig){
+        this.connectConfig = connectConfig;
+        channel = clientUtils.getChannel(connectConfig);
```

Comment:
If the "channel" is already connected, better to shutdown it before this line.

Reply:
ok, will update
if(this.channel != null){
            this.channel.shutdownNow();
       }